.github/build.yaml.gomplate

```diff
@@ -501,5 +479,50 @@ jobs:
           path: |
             *.vhd
           if-no-files-found: error
+
+    {{{ if $config.publishing_pipeline }}}
+
+  publish-vanilla-ami-{{{ $flavor }}}:
```

Comment:
do we really need to push the raw image per flavor? AFAIK we only use the recovery part of it, are the recoveries different between flavors?

Reply:
I say this, because we should push ubuntu/fedora images so we should remove the `skip_images_flavor: ["fedora","ubuntu"]` but that would make the vanilla upload 3 times

Reply:
See this is only active for `$config.publishing_pipeline` which is only active for `master.yaml` and  `master.yaml` also includes 

```yaml
skip_tests_flavor: ["fedora","ubuntu"]                                          
skip_images_flavor: ["fedora","ubuntu"] 
```

which causes it only to be computed per  `opensuse`

Reply:
yeah, but we would need to publish ami images for both systems, which means it will trigger the vanilla image upload 3 times

Reply:
Ok I see, you suggest to name it without a flavor

Reply:
What about now, it is statically forced for opensuse, it is named without a flavor and it only applies for master workflow.